recirq/qml_lfe/learn_dynamics_q.py

```diff
@@ -0,0 +1,238 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Draw two copy data from 1D random scrambling or tsym style circuits.
+
+learn_dynamics_q.py --n=6 --depth=5 --n_data=20 --batch_size=5 --n_shots=1000
+
+Will create 40 total circuits, 20 scrambling and 20 tsym circuits all on
```

Comment:
nit:  I would rearrange this to:  `Will create 20 scrambling and 20 tsym circuits (40 total circuits)` to emphasize the connection to the n_data parameter.  When I first read this, I was momentarily confused why it was 40 circuits but n_data=20.

Reply:
Done.